packages/types/lib/usage/dto.ts

```diff
@@ -0,0 +1,11 @@
+export interface CreateUsageDto {
```

Comment:
If that's a subset of DBUsage can you use pick instead? so types stays in sync

```suggestion
export type CreateUsageDto  = Pick<account_id>;
```

Reply:
Not the hugest fan, and actually I'd make it camelCase for DTOs at least. Well, I'm not sure our API is consistent with casing anyway.